tests/NRedisStack.Tests/RedisFixture.cs

```diff
@@ -18,7 +59,42 @@ public RedisFixture()
             AsyncTimeout = 10000,
             SyncTimeout = 10000
         };
-        Redis = Connect(clusterConfig, out isOSSCluster);
+
+        if (redisEndpointsPath != null && File.Exists(redisEndpointsPath))
+        {
+            string json = File.ReadAllText(redisEndpointsPath);
+            var parsedEndpoints = JsonSerializer.Deserialize<Dictionary<string, EndpointConfig>>(json);
+
+            redisEndpoints = parsedEndpoints ?? throw new Exception("Failed to parse the Redis endpoints configuration.");
+        }
+        else
```

Comment:
Just an idea, would it work to have a default JSON file to load, so we don't do any config from code?

Reply:
Yes, we can replace it with a default JSON file. I did it only to keep it backward compatible with the existing GitHub workflows.